locales/en/defaultTasks.json

```diff
@@ -1,6 +1,6 @@
 {
   "defaultHabit1Text": "1h Productive Work",
-  "defaultHabit1Notes": "When you create a new Habit, you can click the Edit icon and choose for it to represent a positive habit, a negative habit, or both. For some Habits, like this one, it only makes sense to gain points.",
+  "defaultHabit1Notes": "When you create a new Habit, you can click the Edit icon and choose for it to represent a positive Habit, a negative Habit, or both. For some Habits, like this one, it only makes sense to gain points.",
```

Comment:
This is talking about representing real life habits, so I think it can stay lowercase.